doc/users/next_whats_new/2018_01_11_imshow-rgb-scaling-clipping.rst

```diff
@@ -0,0 +1,7 @@
+`Axes.imshow` clips RGB values to the valid range
+-------------------------------------------------
+
+When `Axes.imshow` is passed an RGB or RGBA value with out-of-range
+values, it now issues a warning and clips them to the valid range.
+The old behaviour, wrapping back in to the range, often hid outliers
+and made interpreting RGB images unreliable.
```

Comment:
Should we clarify what ranges are, or perhaps link to an explanation of the ranges?

Reply:
The ranges are explained by the docs for `Axes.imshow`, and the reference to it here is turned into a link by the backticks IIRC.

Reply:
Sorry to drag this out.  I think this really needs to be in the API changes as well in case smeone was depending on the wrap behaviour:

```
Log of changes to Matplotlib that affect the outward-facing API. If updating Matplotlib breaks your scripts, this list may help you figure out what caused the breakage and how to fix it by updating your code.
```